Atomic/info.py

```diff
@@ -66,20 +66,29 @@ def __init__(self):
         super(Info, self).__init__()
         self.beu = BackendUtils()
 
+    def version_tty(self):
+        if self.args.debug:
+            util.write_out(str(self.args))
+        util.write_out(self.version())
```

Comment:
Oh, I see, so we can capture it more easily in the unit test.   I think I probably would have just tested the JSON in the unit tests, and done screen-scraping (i.e. exec `atomic` as a subprocess) for the tty output.

But this is OK too.